### DISCUSSION TITLE
脚注数目限制

### DISCUSSION THREAD

**Discussion by Heloisex** [tuna/thuthesis#742] [Q&A] [answered]
脚注每页重新编号，编号最多到10，理论上**每页**可以有10个脚注；
但是试下来，似乎限制的是**每章/每个tex文件**最多10个脚注，多了就会报错

这个限制是不是有些严格了，还是说操作中做错了什么
小白求问，目前有什么办法可以改吗？

附：说明文档
![image](https://user-images.githubusercontent.com/55721346/158814624-004ddb05-819d-49b3-8606-f5dc6a7a7040.png)
报错（其实已经很多页了，才到第10个脚注，编号并没有超10）：
![image](https://user-images.githubusercontent.com/55721346/158815160-8a1e1747-eca0-4cd2-b295-b94764660d62.png)

**Comment by zepinglee** [comment, marked as answer]
> 脚注每页重新编号，编号最多到10，理论上**每页**可以有10个脚注； 但是试下来，似乎限制的是**每章/每个tex文件**最多10个脚注，多了就会报错

这个似乎是由于依赖的 `footmisc` 宏包更新后导致的。如果每页内的脚注未超过 10 个，但一章内的脚注超过 10 个时，第一次编译会报错，但是第二次编译就正常了。所以改成 warning 已经就能顺利编译了。

**Reply by Heloisex** [reply]
感谢帮助~
理论上似乎是的，不过不知道为什么，我这边报错之后，编译器就卡在那里了……
btw，
刚刚试了下把thuthesis.cls和.dtx里面的“ >10”粗暴调大了些，也还挺好用的 ：）
第一次编译时，>10的部分因为footmisc包里面字体的问题没有序号；第二次编译就完全正常了

**Reply by zepinglee** [reply]
你试一下 1707d1d 可以正常编译吗？
